src/main/java/org/frc5687/powerup/robot/subsystems/Carriage.java

```diff
@@ -77,7 +77,7 @@ protected double returnPIDInput() {
 
     @Override
     protected void usePIDOutput(double output) {
-        DriverStation.reportError("Arm PID output at " + output, false);
+        SmartDashboard.putNumber("Arm PID output at ", output);
```

Comment:
You are using duplicate keys, so the Arm and Carriage outputs are overwriting each other. 

I'd suggest using `Carriage/PID/output`